src/platform/tls_schannel.c

```diff
@@ -516,6 +511,11 @@ typedef struct QUIC_ACH_CONTEXT {
     //
     UNICODE_STRING BlockedChainingModes[1];
 
+    //
+    // CredConfig certificate hash used to find the server certificate.
+    //
+    SCHANNEL_CERT_HASH_STORE CertHash[0];
```

Comment:
doesn't this collide with the BlockedChainingModes array above?

Reply:
No. That really is just an array of 1 item. Not a variable sized array.